entity-framework/efcore-and-ef6/porting/port-database.md

```diff
@@ -35,3 +35,5 @@ You may want to consider an alternate workflow and follow steps similar to the [
 ## Customize the code generation
 
 As of EF Core 6, it is possible to customize the code generated by scaffolding by using [T4 templates](/visualstudio/modeling/code-generation-and-t4-text-templates). See [T4 and EF Core Reverse Engineering](https://www.bricelam.net/2020/02/03/t4-and-efcore.html) for details.
```

Comment:
"As of EF Core 6, it is possible to customize..." It is currently **not** possible.